app/controllers/mentees_controller.rb

```diff
@@ -16,4 +16,13 @@ def add_course
     Assign.mentor(current_mentee, course_id)
     render json: current_mentee, status: :ok
   end
+
+  def fetch_mentee_record
+    current_mentee = Mentee.find_by(:id => params[:id])
+    if current_mentee.present?
+      render json: { message: "Mentee Record fetched succesfully", "Mentee": current_mentee.user, "Mentor": current_mentee.mentor.user, "Course": current_mentee.course }, status: 200
```

Comment:
For consistency, it would be better to name the keys the same way (with lowercase):

```rb
json: {
  message: 'Success message',
  mentee: current_mentee.user,
  mentor: current_mentee.mentor.user,
  course: current_mentee.course
}
```